codestodians/komlie-frontend
# [Feature] Activate cart functionality/feature

#### What does this PR do?
- Allows the addition and removal of items to/from cart.

#### Description of Task to be completed?
- Add relevant queries and mutations
- Update logic to account for addition and removal

#### How should this be manually tested?
- Check out to the `feature/activate-cart-functionality-167043851` branch
- Go to the `product details` page
- Click on the `Add to cart` button
- The item should be successfully added
- Go to the cart
- Remove an item with the minus button/icon
- The item should be successfully removed

#### Any background context you want to provide?
- There's still work to be done on the backend
- Of the work to be done on the backend, the addition of `quantity` to the product-cart model might be useful. This is because currently, a product can be added only once.

#### What are the relevant pivotal tracker stories?
- [167043851](https://www.pivotaltracker.com/story/show/167043851)

#### Extra details
- On the list of products page, there's the possibility of adding a button or icon to the products to also add them to cart directly. However, I opted to make that another story (In the event that we decide on it). This is because it was already getting long and arduous for me and I need this closed out.
- As usual, please forgive the simplistic look. I need that functionality and then I can modify the look/outlook in subsequent stories
